daemon/logger/loginfo.go

```diff
@@ -127,3 +129,45 @@ func (info *Info) ImageFullID() string {
 func (info *Info) ImageName() string {
 	return info.ContainerImageName
 }
+
+// InfoMap returns information about a container as a map.
+func (info *Info) InfoMap(keyMod func(string) string) map[string]string {
```

Comment:
I'm not sure I understand the purpose of this new function.
Why wouldn't we just set the keys as needed in the logger?